Dockerfile

```
@@ -158,44 +170,37 @@ RUN rm /etc/apache2/sites-enabled/000-default.conf
 COPY --from=builder /tmp/local/ /opt/perl/local/
 ENV PERL5LIB="/opt/product-opener/lib/:/opt/perl/local/lib/perl5/"
 ENV PATH="/opt/perl/local/bin:${PATH}"
+# Create writable dirs and change ownership to www-data
+RUN \
+    mkdir -p var/run/apache2/ && \
+    chown www-data:www-data var/run/apache2/ && \
+    for path in data html_data users products product_images orgs new_images logs tmp; do \
+        mkdir -p /mnt/podata/${path}; \
+    done && \
+    chown www-data:www-data -R /mnt/podata && \
+    # Create symlinks of data files that are indeed conf data in /mnt/podata (because we currently mix data and conf data)
+    for path in ecoscore emb_codes forest-footprint ingredients lang packager-codes po taxonomies templates; do \
```

```suggestion
    for path in ecoscore emb_codes forest-footprint ingredients lang packager-codes po t taxonomies templates; do \
```

We also need the /t directory because some tests need to read expected responses from the /t/expected_test_results / dir.

---

Otherwise this fails:

docker-compose run --rm  backend perl /opt/product-opener/t/packaging.t

---

why do we seek it under $data_root instead of current dir, or relative to test file ?

---

Done with  __FILE__